cmd/admin.go

```diff
@@ -1,4 +1,5 @@
 // Copyright 2016 The Gogs Authors. All rights reserved.
+// Copyright 2016 The Gitea Authors. All rights reserved.
```

Comment:
@lunny on Golang project I don't know. In PHP and Python projects we put on year desc.